components/esp_system/panic.c

```diff
@@ -373,6 +373,27 @@ void esp_panic_handler(panic_info_t *info)
     wdt_hal_write_protect_enable(&rtc_wdt_ctx);
 #if CONFIG_ESP_SYSTEM_PANIC_PRINT_REBOOT || CONFIG_ESP_SYSTEM_PANIC_SILENT_REBOOT
 
+#if CONFIG_ESP_SYSTEM_PANIC_REBOOT_DELAY_SECONDS
+
+    disable_all_wdts();
+
+    panic_print_str("Waiting to reboot...\r\n");
+
+    for(int i = 0; i < CONFIG_ESP_SYSTEM_PANIC_REBOOT_DELAY_SECONDS; i++){
```

Comment:
Please note our [style guide](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/contribute/style-guide.html#horizontal-space):
`for (int i = 0; i < CONFIG_ESP_SYSTEM_PANIC_REBOOT_DELAY_SECONDS; i++) {`

Reply:
updated.